includes/admin/post-types/meta-boxes/class.llms.meta.box.coupon.php

```diff
@@ -157,7 +157,8 @@ public function get_fields() {
 						'multi' => true,
 						'selected' => $c ? $c->get_array( 'coupon_courses' ) : array(),
 						'data_attributes' => array(
-							'post-type' => 'course',
+							'post-type'     => 'course',
```

Comment:
add version changelogs to updated methods and file header docblocks

Reply:
fixed in https://github.com/gocodebox/lifterlms/pull/832/commits/bc7ca7152db7783d7ca1b91c4471ed8389acb90a